hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -842,9 +537,9 @@ public boolean renameIdempotencyCheckOp(
 
     // removing isDir from debug logs as it can be misleading
     LOG.debug("rename({}, {}) failure {}; retry={} etag {}",
-              source, destination, op.getResult().getStatusCode(), op.isARetriedRequest(), sourceEtag);
+        source, destination, op.getResult().getStatusCode(), op.isARetriedRequest(), sourceEtag);
```

Comment:
diffs can be reduced for spacing

Reply:
This was fixed by IDE. I think this is only right indentation. We can have a few diffs if we are rectifying some thing IMO. What do you think??